repository/DeleteUserRepo/DeleteUserRepo.go

```diff
@@ -1,8 +0,0 @@
-package DeleteUserRepo
-import(
-	"example/persons/db"
-	"example/persons/models"
-)
-func DeleteUserRepo(alpha *[]models.Person,id int)error{
```

Comment:
try to give meaningful names for variables.
variable names like alpha can be confusing in terms for bigger projects.
suggestion: person